src/js/browser-plugin.js

```diff
@@ -1355,7 +1349,7 @@ function openClaimPopup(tweet) {
     confirmButtonColor: buttonColor,
     confirmButtonText: browserAPI.i18n.getMessage('submit'),
     focusConfirm: true,
-    onClose: sendCloseClaimLog(node),
+    onClose: sendCloseClaimLog,
```

Comment:
If we do like this, then when the "sendCloseClaimLog" function will be executed, the parameters will be empty. I think it is better to just copy the function code here: like, for example, how it is done in the "preConform" action in the popup opened on line 1353.